CocoaLumberjack.podspec

```diff
@@ -24,22 +24,16 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target    = '9.0'
 
-  s.default_subspecs = 'Default', 'Extensions'
+  s.module_map = 'Framework/Lumberjack/CocoaLumberjack.modulemap'
+  s.default_subspecs = 'Default'
```

Comment:
Good! Then I'm closing this PR in first. About another PR..., please wait for a little while? (probably at the longest less than 1 week though)
Thanks!